src/components/TaskItem.module.css

```
@@ -0,0 +1,107 @@
+  text-align: left;
+  line-height: 1.4;
+}
+
+.checkbox {
+  flex-shrink: 0;
+  appearance: none;
+  border-radius: var(--_radius);
+  width: 1em;
+  height: 1em;
+  background-color: hsl(var(--muted));
+  border-radius: var(--_radius);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border-radius is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)